app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -1,76 +1,82 @@
 
-.notice-summary{role: "group", "aria-labelledby": "notice-summary-heading", tabindex:"-1", style: "display: none;"}
-  %h1.heading-medium.notice-summary-heading{id: "notice-summary-heading"}
+.notice-summary{ role: 'group', 'aria-labelledby': 'notice-summary-heading', tabindex: '-1', style: 'display: none;' }
+  %span.heading-medium.notice-summary-heading{ id: 'notice-summary-heading' }
     %span.msg
 
-.error-summary{role: "group", "aria-labelledby": "error-summary-heading", tabindex: "-1", style: "display: none;"}
-  %h1#error-summary-heading.heading-medium.error-summary-heading
+.error-summary{ role: 'group', 'aria-labelledby': 'error-summary-heading', tabindex: '-1', style: 'display: none;' }
+  %span#error-summary-heading.heading-medium.error-summary-heading
     %span.msg
 
-.grid-row#api-key{:data => {api_key: current_user.api_key}}
+#api-key.grid-row{ data: { api_key: current_user.api_key } }
   .column-two-thirds
-    = render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
+    = render partial: 'layouts/header', locals: { page_heading: t('.page_heading') }
   .column-one-third.scheme-filters
     -# FORM
     -# AGFS / LGFS Radios
     = render partial: 'scheme_filters'
 
 = form_tag case_workers_admin_allocations_path, method: :get do
   = hidden_field_tag :tab, params[:tab]
-  = form_for [:case_workers, :admin, @allocation] do |f|
-    = hidden_field_tag :tab, params[:tab]
-    = hidden_field_tag :scheme, params[:scheme]
-    = hidden_field_tag :filter, params[:filter] if params[:filter]
-    = hidden_field_tag :page, params[:page]
-    = hidden_field_tag :value_band_id, params[:value_band_id] if params[:value_band_id]
+  = hidden_field_tag :scheme, params[:scheme]
+  = hidden_field_tag :filter, params[:filter] if params[:filter]
+  = hidden_field_tag :page, params[:page]
+  = hidden_field_tag :value_band_id, params[:value_band_id] if params[:value_band_id]
 
-    %h2.heading-medium
-      = t('.allocate_claims')
+  %fieldset.inline
+    %legend
+      %span.heading-medium
+        = t('.allocate_claims')
     .grid-row.js-typeahead
       .column-one-third.js-typeahead
         = label_tag :quantity_to_allocate, t('.no_of_claims'), class: 'form-label-bold'
-        = number_field_tag :quantity_to_allocate, nil, { style: 'width:100%', min: 0, size: 5, class: 'form-control'}
+        = number_field_tag :quantity_to_allocate, nil, { style: 'width: 100%', min: 0, size: 5, class: 'form-control' }
       .column-one-third
         #cc-caseworker
-          = f.label :case_worker_id, t('dictionary.models.case_worker'), for: "allocation_case_worker_id_autocomplete", class: 'form-label-bold'
-          = f.collection_select :case_worker_id, @case_workers, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete' , style: 'width:100%'}
+          = label_tag :allocation_case_worker_id, t('dictionary.models.case_worker'), class: 'form-label-bold'
```

Comment:
**NOT a blocker** 
I remember looking at this with you. You should be able to use `form_with` like this

```
form_with(model: [:case_workers, :admin, @allocation]) do
  = f.hidden_field :tab, params[:tab]
  ....
  = f.label :case_worker_id, t('dictionary.models.case_worker'), class: 'form-label-bold'
  = f.collection_select :case_worker_id, @case_workers, :id, :name, { include_blank: ''.html_safe }, { class: 'form-control fx-autocomplete', style: 'width: 100%' }
```

note the removal of `allocation` from the field name if you do this

an alternative might be:
```
form_with(url: case_workers_admin_allocations_path, method: :get) do
 ```

Reply:
👍 , I will update all forms in a later sprint to use `form_with`

Reply:
👍